model/variable.h

```diff
@@ -167,6 +167,9 @@ namespace minsky
     bool isStock() const {return type()==stock || type()==integral;}
 
     virtual ~VariableBase();
+    
+    bool varTabDisplay=false;
+    void toggleVarTabDisplay() {varTabDisplay=!varTabDisplay;}     
```

Comment:
You should overrride attachedToDefiningVar() here, and return the value of varTabDisplay. This will simplify the Wire::attachedToDefiningVar() implementation later.

Reply:
Done, thanks.  But struggling to get all the items to disappear /reappear when I toggleVarTabDisplay()